pkg/controller/secret/secret_controller.go

```diff
@@ -491,25 +478,71 @@ func removeConfigFromAlertManager(r *ReconcileSecret, request *reconcile.Request
 }
 
 // generateRoutes generates a set of AlertManger routes based off of configured constants.
-func generateRoutes(warningSlice, nullSlice []string) []*alertmanager.Route {
+func generateRoutes() []*alertmanager.Route {
 	var routes []*alertmanager.Route
 
-	for _, alert := range warningSlice {
+	// Use the following to make critical alerts "warning"
+	{
+		var rec = "make-it-warning"
+
+		// https://issues.redhat.com/browse/OSD-1922
 		routes = append(routes,
 			&alertmanager.Route{
-				Receiver: "make-it-warning",
+				Receiver: rec,
 				Match: map[string]string{
-					"alertname": alert,
+					"alertname": "KubeAPILatencyHigh",
+					"severity":  "critical",
 				},
 			})
 	}
```

Comment:
Why is this block wrapped with `{}`?

Reply:
To scope the `rec` variable and ensure we don't accidentally use the wrong value for Receiver for Routes we expect to be going to `null`.  Originally was flat and two vars but I could see future revisions with copy and paste landing the wrong value.